test/integration/targets/setup_flatpak_remote/tasks/main.yaml

```diff
@@ -0,0 +1,12 @@
+- name: Set up dummy flatpak repository remote
+  block:
+
+    - name: Copy repo into place
+      copy:
+        src: repo
```

Comment:
Copying so many files with the `copy` module isn't very efficient. Also, adding so many files to the test adds overhead we could do without. Perhaps it would be better to commit all the repo files in a single tgz or zip archive instead.

Reply:
Good point! I will test this.

Reply:
Single tgz sounds good